.github/workflows/model_uploader.yml

```diff
@@ -99,55 +99,55 @@ jobs:
       verified_license_line: ${{ steps.init_license_line.outputs.verified }}
       unverified_license_line: ${{ steps.init_license_line.outputs.unverified }}
 
-  # Step 3: Check if the model already exists in the model hub
-  checking-out-model-hub:
-    needs: init-workflow-var
-    runs-on: 'ubuntu-latest'
-    permissions:
-      id-token: write
-      contents: read
-    environment: opensearch-py-ml-cicd-env
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v3
-      - name: Set Up Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: '3.x'
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v2
-        with:
-          aws-region: ${{ secrets.MODEL_UPLOADER_AWS_REGION }}
-          role-to-assume: ${{ secrets.MODEL_UPLOADER_ROLE }}
-          role-session-name: checking-out-model-hub
-      - name: Check if TORCH_SCRIPT Model Exists
-        if: github.event.inputs.tracing_format == 'TORCH_SCRIPT' || github.event.inputs.tracing_format == 'BOTH'
-        run: |
-          TORCH_FILE_PATH=$(python utils/model_uploader/save_model_file_path_to_env.py \
-              ${{ needs.init-workflow-var.outputs.sentence_transformer_folder }} ${{ github.event.inputs.model_id }} \
-              ${{ github.event.inputs.model_version }} TORCH_SCRIPT)
-          aws s3api head-object --bucket ${{ secrets.MODEL_BUCKET }} --key $TORCH_FILE_PATH > /dev/null 2>&1 || TORCH_MODEL_NOT_EXIST=true
-          if [[ -z $TORCH_MODEL_NOT_EXIST ]]
-          then
-            echo "${{ github.event.inputs.model_id }} already exists on model hub for TORCH_SCRIPT format and ${{ github.event.inputs.model_version }} version."
-            exit 1
-          fi
-      - name: Check if ONNX Model Exists
-        if: github.event.inputs.tracing_format == 'ONNX' || github.event.inputs.tracing_format == 'BOTH'
-        run: |
-          ONNX_FILE_PATH=$(python utils/model_uploader/save_model_file_path_to_env.py \
-            ${{ needs.init-workflow-var.outputs.sentence_transformer_folder }} ${{ github.event.inputs.model_id }} \
-            ${{ github.event.inputs.model_version }} ONNX)
-          aws s3api head-object --bucket ${{ secrets.MODEL_BUCKET }} --key $ONNX_FILE_PATH > /dev/null 2>&1 || ONNX_MODEL_NOT_EXIST=true
-          if [[ -z $ONNX_MODEL_NOT_EXIST ]]
-          then
-            echo "${{ github.event.inputs.model_id }} already exists on model hub for ONNX format and ${{ github.event.inputs.model_version }} version."
-            exit 1
-          fi
+  # # Step 3: Check if the model already exists in the model hub
```

Comment:
I'm just wondering if we can add a flag in the workflow UI. 
By default, it will check out the model hub for duplication, but if we uncheck it won't check model hub for duplication. In that way, we will have control in our end if we want to make any changes? What do you think? That's not urgent, we can do that later too.

Reply:
Not sure if that's a good idea because we should not overwrite model files generally.

Reply:
Yeah, which is why we mark this as checked by default.

Reply:
I think this is reasonable for checking by default. Seems add a flag to control check or not is more flexible. I prefer to add a flag